partner_capital/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Partners Capital
+================
+
+This module extends the functionality of partners to support setting the
+country of origin of capital, registered capital amount and turnover.
```

Comment:
On code "registered capital" was renamed as "Capital amount", maybe could be better to update also in README